components/KeywordLink/KeywordLink.js

```diff
@@ -0,0 +1,14 @@
+import Link from 'next/link';
+
+import HashIcon from '../../public/img/hashicon.svg';
+
+export const KeywordLink = (prop) => (
+  <div className="block py-1 md:inline md:px-5 md:first:pl-0 text-accent-500">
+    <HashIcon className="w-3 inline mr-3 -mt-1" />
```

Comment:
I am not sure what is the design of this place. But why is the negative margin needed?

Reply:
It is for the svg icon - the icon is aligned on the center of the text not on the baseline. But if I use the baseline align, it looks strange too. So I made it "by hand".